code/modules/cargo/packs/engineering.dm

```diff
@@ -334,3 +334,13 @@
 	contains = list(/obj/machinery/power/portagrav = 1)
 	crate_name = "portable gravity unit crate"
 	crate_type = /obj/structure/closet/crate/engineering
+
+/datum/supply_pack/engineering/golfcart
+	name = "Golf Cart Parts Kit Crate"
+	desc = "Contains the parts to build a cart intended for moving heavy machinery and cargo across the station. \
+		Nanotrasen assumes no liability for carts operated as 'party wagons'."
+	cost = CARGO_CRATE_VALUE * 5
```

Comment:
Up the price, this is dirt cheap for what it offers. At least triple what it is now feels a bit more reasonable, more of an investment